.github/workflows/check-pr-title.yaml

```diff
@@ -9,6 +13,8 @@ on:
 
 jobs:
   check-pr-title:
+    permissions:
+      pull-requests: read
```

Comment:
As per https://github.com/aslafy-z/conventional-pr-title-action/issues/260, it seems that `conventional-pr-title-action` requires `statuses: write` permission

Reply:
Updated.